core/promise/dto.go

```diff
@@ -27,9 +27,9 @@ type SignedPromise struct {
 
 // Promise represents payment promise between two parties
 type Promise struct {
-	SerialNumber int
-	IssuerID     string
-	BenefiterID  string
+	SerialNumber int    `storm:"id"`
```

Comment:
I really really recommend to start using Promise structures from payments lib, as these promises will land on contract anyway, and a lot of work will need to be done to align them with payment contract expected structure later

Reply:
Looks like it's out of the scope of this PR. 
Created issue for changing it in the same milestone. https://github.com/mysteriumnetwork/node/issues/454